tools/update_apt_archive

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright 2019 The Project Buendia Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at: http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distrib-
+# uted under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
+# specific language governing permissions and limitations under the License.
+
+set -e
+
+if [ "$1" = "-h" -o -z "$1" -o -z "$2" -o -z "$3" ]; then
+    echo "Usage: $0 <repository-dir> <branch> <artifact-dir>"
```

Comment:
It looks like $1 is supposed to be the `packages` subdirectory of the repo (as in `builds/packages`), not the repo directory itself (`builds`)—could the help message make this a tad clearer?